guide: update how to share many exps (#3593)
iterative/dvc.org: content/docs/user-guide/experiment-management/cleaning-experiments.md

    cnn-64
    cnn-128
$ dvc exp remove cnn-32 cnn-64
Removed experiments: cnn-32,cnn-64
```

> ## Removing multiple experiments
> 
> You may wish to remove multiple experiments at once. For example, if you are
> finished experimenting on the current commit and want to discard all experiments
> based on that commit, use `--rev HEAD`.
> 
> ```
> $ dvc exp list

Reviewer: Should we rename the title since it's a relatively narrow feature?

````suggestion
## Removing multiple experiments (based on parent commit)

You may wish to remove multiple experiments at once. For example, if you are
finished experimenting on the current Git commit and want to discard all
experiments derived from it, use `dvc exp remove --rev` with `HEAD`.

```dvc
$ dvc exp list
````

```dvc
$ dvc exp gc --workspace

> WARNING: This will remove all experiments except those derived from the workspace of the current repo. Run queued experiments will be removed.
> Are you sure you want to proceed? [y/n]: y
> Removed 5 experiments. To remove unused cache files use 'dvc gc'.

Author: ```suggestion

```

> $ dvc exp gc --workspace
> WARNING: This will remove all experiments except those derived from the workspace of the current repo. Run queued experiments will be removed.
> Are you sure you want to proceed? [y/n]: y
> Removed 5 experiments. To remove unused cache files use 'dvc gc'.
> $ dvc exp list --all

Reviewer: Not sure we need so much detail in the sample output, could just be an empty line (or `...`). Conveys the idea without noise

```suggestion
$ dvc exp gc --workspace

$ dvc exp show --all-tags
```

```dvc
$ dvc exp gc --all-tags

> WARNING: This will remove all experiments except those derived from the workspace of the current repo. Run queued experiments will be removed.

Reviewer 2: `except those derived from the workspace of the current repo.`
->
`except those derived from any Git tags of the current repo`

> WARNING: This will remove all experiments except those derived from the workspace of the current repo. Run queued experiments will be removed.

Author: ```suggestion
WARNING: This will remove all experiments except those derived from any tags of the current repo. Run queued experiments will be removed.
```

> $ dvc exp gc --all-tags
> WARNING: This will remove all experiments except those derived from any tags of the current repo. Run queued experiments will be removed.
> Are you sure you want to proceed? [y/n]: y
> Removed 2 experiments. To remove unused cache files use 'dvc gc'.
> $ dvc exp show --all-tags

Reviewer: Same

```suggestion
$ dvc exp gc --all-tags

$ dvc exp show --all-tags
```

### Deleting Experiment-Related Objects in DVC Cache

> Note that `dvc exp gc` and `dvc exp remove` don't delete any objects in the DVC
> <abbr>cache</abbr>. In order to remove the cache objects, e.g. model files,

Reviewer: CE (committing)
```suggestion
Note that `dvc exp gc` and `dvc exp remove` do not delete any objects in the
DVC <abbr>cache</abbr>. In order to remove the cache objects, e.g. model files,
```

## Removing experiments from remotes

As you push the experiments with `dvc exp push`, Git remotes may be become
cluttered with experiment references. To remove experiments from a Git remote,
use `dvc exp remove -g`.

> ```

Reviewer: ````suggestion
```dvc
````

> dvc exp remove -g origin exp-ab780

Reviewer: ```suggestion
$ dvc exp remove -g origin exp-ab780
```
